lib/litestream.rb

```diff
@@ -68,7 +69,8 @@ def queue
     def replicate_process
       info = {}
       if !`which systemctl`.empty?
-        systemctl_status = `systemctl status litestream`.chomp
+        command = Litestream.configuration&.systemctl_status_command || "systemctl status litestream"
+        systemctl_status = `#{command}`.chomp
```

Comment:
When you move this to an accessor on the module, I want the default to live in the getter so we can do:
```suggestion
        systemctl_status = `#{Litestream.systemctl_command}`.chomp
```

Reply:
Done!